core/test/financial_templates/Liquidatable.js

```diff
@@ -256,7 +292,18 @@ contract("Liquidatable", function(accounts) {
         assert.equal(liquidation.disputer, disputer);
         assert.equal(liquidation.disputeTime.toString(), disputeTime.toString());
       });
-      it("Liquidation has already been disputed", async () => {
+      it("Dispute initiates an oracle call", async () => {
+        const disputeTime = await liquidationContract.getCurrentTime();
+        await liquidationContract.dispute(liquidationParams.uuid, sponsor, { from: disputer });
+        // Oracle should have an enqueued price after calling dispute
+        const pendingRequests = await mockOracle.getPendingQueries();
+        assert.equal(
+          pendingRequests[0]["identifier"],
```

Comment:
I think I realized why we've never had to do this padding: we generally compare in utf8 space, not hex. So, you should be able to get around the padding by:
```
const { hexToUtf8 } = web3.utils; // put this somewhere way up top :)
assert.equal(hexToUtf8(pendingRequests[0]["identifier"]), hexToUtf8(priceTrackingIdentifier));
```

The nice consequence of this is that when you get failures, you see the human readable string and not the hex.

Reply:
ok great! will update the test :)